Sources/MapLibreSwiftUI/Examples/Layers.swift

```diff
@@ -76,6 +88,44 @@ let pointSource = ShapeSource(identifier: "points") {
     .ignoresSafeArea(.all)
 }
 
+#Preview("Clustered Circles with Symbols") {
+    @State var camera = MapViewCamera.center(
+        CLLocationCoordinate2D(latitude: 48.2082, longitude: 16.3719),
+        zoom: 5,
+        direction: 0
+    )
+    return MapView(styleURL: demoTilesURL, camera: $camera) {
+        // Clusters pins when they would touch
+
+        // Cluster == YES shows only those pins that are clustered, using .text
+        CircleStyleLayer(identifier: "simple-circles-clusters", source: clustered)
+            .radius(16)
+            .color(.systemRed)
+            .strokeWidth(2)
+            .strokeColor(.white)
+            .predicate(NSPredicate(format: "cluster == YES"))
+
+        SymbolStyleLayer(identifier: "simple-symbols-clusters", source: clustered)
+            .textColor(.white)
+            .text(expression: NSExpression(format: "CAST(point_count, 'NSString')"))
+            .predicate(NSPredicate(format: "cluster == YES"))
+
+        // Cluster != YES shows only those pins that are not clustered, using an icon
+        CircleStyleLayer(identifier: "simple-circles-non-clusters", source: clustered)
```

Comment:
💯 thanks for including an example for this. This exact case trips up so many people ;)